cms/templates/widgets/header.html

```diff
@@ -189,13 +190,13 @@ <h2 class="info-course">
 
     <div class="wrapper wrapper-r">
       % if user.is_authenticated():
```

Comment:
Can we move the user authentication check into the static library?  (assuming it makes sense)